pyvista/core/utilities/features.py

```diff
@@ -14,7 +14,37 @@
 from .helpers import wrap
 
 
-def voxelize(mesh, density=None, check_surface=True):
+def padded_bins(mesh, density):
```

Comment:
Is this intended for general use, or should we mark it as private?
```suggestion
def _padded_bins(mesh, density):
```

Reply:
Thank you, good change.